src/sentry/tasks/post_process.py

```diff
@@ -141,6 +142,13 @@ def post_process_group(event, is_new, is_regression, is_sample, is_new_group_env
                             event=event,
                         )
 
+        if is_new:
+            process_resource_change_bound.delay(
+                action='created',
+                sender='Group',
+                instance_id=event.group.id,
```

Comment:
I'd normally suggest using `event.group_id`, but it looks like we already explicitly bind it above. I'd still recommend doing that just incase something changes in the future.